app/src/main/java/org/fossasia/phimpme/gallery/activities/SingleMediaActivity.java

```diff
@@ -929,16 +942,30 @@ public void onClick(DialogInterface dialog, int which) {
 
                     }
                 });
-                
+
                 descriptionDialog.getButton(AlertDialog.BUTTON_POSITIVE).setOnClickListener(new View.OnClickListener() {
                     @Override
                     public void onClick(View v) {
                         descriptionDialog.dismiss();
                         voiceInput = editTextDescription.getText().toString();
                         if (temp == null) {
                             databaseHelper.addImageDesc(new ImageDescModel(pathForDescription, editTextDescription.getText().toString()));
+                            SnackBarHandler.show(parentView,"Description Saved");
```

Comment:
Please don't hardcode string like this

Reply:
@anantprsd5  Ok ! adding to strings.xml and updating